components/code_preview.tsx

```diff
@@ -69,7 +86,7 @@ export default class CodePreview extends React.PureComponent {
         }
     }
 
-    handleReceivedCode = (data) => {
+    handleReceivedCode = (data: any) => {
```

Comment:
was it not possible to figure out the type of data here, We want to avoid any as much as possible.

Reply:
Hi @AshishDhama, the correct type here should be **string**, But the we are accessing property **nodeName** from data which does not exist on type string. 
So neither can we give it type **Node** nor **string**

Reply:
resolved